harness/libraries/status_file.py

```diff
@@ -851,6 +895,8 @@ def get_status_info(test_id, event_type, event_subtype,
         test_instance_info['test'],
         test_instance_info['test_id'], apptest_layout.test_build_dirname)
 
+    test_instance_info['machine'] = os.environ['RGT_MACHINE_NAME'] if 'RGT_MACHINE_NAME' in os.environ else no_value
```

Comment:
@hagertnl - what happens when we pass `no_value`? are things still findable in influx through some other mechanism?

Reply:
in theory, no. in practice... i hope not 😄